usr/lib/cca_stdll/cca_specific.c

```diff
@@ -3245,7 +3245,7 @@ CK_RV token_specific_set_attrs_for_new_object(STDLL_TokData_t *tokdata,
     CK_ATTRIBUTE *sensitive_attr = NULL;
 #ifndef NO_PKEY
     CK_ATTRIBUTE *pkey_attr = NULL, *ecp_attr = NULL;
-    CK_BBOOL add_pkey_extractable = CK_FALSE;
+    CK_BBOOL add_pkey_extractable = CK_FALSE, bfalse = FALSE;
```

Comment:
Looks like in the past we used CK_FALSE, but you are now consequently using just FALSE (which is the same). So you could even change the CK_FALSE in this line to FALSE, while at it. But yes, this would not directly belong to the fix.